docs/sources/flow/reference/compatibility/_index.md

```diff
@@ -0,0 +1,377 @@
+- [prometheus.exporter.postgres]({{< relref "../components/prometheus.exporter.postgres.md" >}})
+- [prometheus.exporter.process]({{< relref "../components/prometheus.exporter.process.md" >}})
+- [prometheus.exporter.redis]({{< relref "../components/prometheus.exporter.redis.md" >}})
+- [prometheus.exporter.snmp]({{< relref "../components/prometheus.exporter.snmp.md" >}})
+- [prometheus.exporter.snowflake]({{< relref "../components/prometheus.exporter.snowflake.md" >}})
+- [prometheus.exporter.squid]({{< relref "../components/prometheus.exporter.squid.md" >}})
+- [prometheus.exporter.statsd]({{< relref "../components/prometheus.exporter.statsd.md" >}})
+- [prometheus.exporter.unix]({{< relref "../components/prometheus.exporter.unix.md" >}})
+- [prometheus.exporter.vsphere]({{< relref "../components/prometheus.exporter.vsphere.md" >}})
+- [prometheus.exporter.windows]({{< relref "../components/prometheus.exporter.windows.md" >}})
+{{< /collapse >}}
+
+<!-- END GENERATED SECTION: EXPORTERS OF Targets -->
+
+
+<!-- NOTE: this title is used as an anchor in links. Do not change. -->
+### Targets Consumers
+Below are components that _consume_ Targets grouped by namespace. Click
+on the namespace to expand and see more detail.
+
+<!-- START GENERATED SECTION: CONSUMERS OF Targets -->
+
+{{< collapse title="discovery" >}}
+- [discovery.relabel]({{< relref "../components/discovery.relabel.md" >}})
+{{< /collapse >}}
+
+{{< collapse title="local" >}}
+- [local.file_match]({{< relref "../components/local.file_match.md" >}})
+{{< /collapse >}}
+
+{{< collapse title="loki" >}}
+- [loki.source.docker]({{< relref "../components/loki.source.docker.md" >}})
+- [loki.source.file]({{< relref "../components/loki.source.file.md" >}})
+- [loki.source.kubernetes]({{< relref "../components/loki.source.kubernetes.md" >}})
+{{< /collapse >}}
+
+{{< collapse title="otelcol" >}}
+- [otelcol.processor.discovery]({{< relref "../components/otelcol.processor.discovery.md" >}})
+{{< /collapse >}}
+
+{{< collapse title="prometheus" >}}
+- [prometheus.scrape]({{< relref "../components/prometheus.scrape.md" >}})
+{{< /collapse >}}
+
+{{< collapse title="pyroscope" >}}
+- [pyroscope.ebpf]({{< relref "../components/pyroscope.ebpf.md" >}})
+- [pyroscope.scrape]({{< relref "../components/pyroscope.scrape.md" >}})
+{{< /collapse >}}
+
+<!-- END GENERATED SECTION: CONSUMERS OF Targets -->
+
+
+## Prometheus `MetricsReceiver`
+
+The Prometheus metrics are sent between components using `MetricsReceiver`s. 
+`MetricsReceiver`s are [capsules]({{< relref "../../config-language/expressions/types_and_values/#capsules" >}})
+that are exported by components that can receive Prometheus metrics. Components that
+can consume Prometheus metrics can be passed the `MetricsReceiver` as an argument. Use the
+following components to build your Prometheus metrics pipeline:
+
+<!-- NOTE: this title is used as an anchor in links. Do not change. -->
+### Prometheus `MetricsReceiver` Exporters
```

Comment:
This would break the anchors that we link to directly. I'm aware of the TOC, but I thought that it's more important to have good anchors 🤔 We could link to the parent heading though as an alternative...